applications/nrf_desktop/src/modules/Kconfig.ble_latency

```diff
@@ -0,0 +1,44 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig DESKTOP_BLE_LATENCY_ENABLE
+	bool "BLE latency module"
+	depends on DESKTOP_BT_PERIPHERAL
+	help
+	  Enable BLE latency module that controls Bluetooth LE connection
+	  latency. The module lowers the connection latency when configuration
+	  channel is in use or when firmware update is received by SMP (low
+	  latency ensures quick data exchange).
+
+	  The module also keeps the connection latency low for LLPM connections
+	  to improve performance and disconnects Bluetooth peer if connection
+	  has not been secured in the predefined amount of time after the
+	  connection occurred.
+
+if DESKTOP_BLE_LATENCY_ENABLE
+
+config DESKTOP_BLE_SECURITY_FAIL_TIMEOUT_S
```

Comment:
why does this configuration belong to the Latency module? It seems like a bonding (security) module is a better fit for it

Reply:
`NCSDK-19652`